lib/api/src/grpc/proto/collections.proto

```diff
@@ -463,12 +463,58 @@ message GeoIndexParams {
   optional bool on_disk = 1; // If true - store index on disk.
 }
 
+enum Language {
+  UnspecifiedLanguage = 0;
+  Arabic = 1;
+  Azerbaijani = 2;
+  Basque = 3;
+  Bengali = 4;
+  Catalan = 5;
+  Chinese = 6;
+  Danish = 7;
+  Dutch = 8;
+  English = 9;
+  Finnish = 10;
+  French = 11;
+  German = 12;
+  Greek = 13;
+  Hebrew = 14;
+  Hinglish = 15;
+  Hungarian = 16;
+  Indonesian = 17;
+  Italian = 18;
+  Kazakh = 19;
+  Nepali = 20;
+  Norwegian = 21;
+  Portuguese = 22;
+  Romanian = 23;
+  Russian = 24;
+  Slovene = 25;
+  Spanish = 26;
+  Swedish = 27;
+  Tajik = 28;
+  Turkish = 29;
+}
+
+message StopwordsSet {
+  repeated Language languages = 1; // List of languages to use stopwords from
+  repeated string custom = 2; // List of custom stopwords
+}
+
+message StopwordsInterface {
+  oneof stopwords {
+    Language language = 1; // Single language stopwords
+    StopwordsSet set = 2; // Multiple languages and custom stopwords
+  }
+}
+
```

Comment:
`StopwordsInterface` is nice for REST, but too complex for gRPC. It will significantly increase complexity of client-side code and won't provide any meaningful UX improvements. I propose to use `StopwordsSet` directly and implement proper `Builders` in clients, so we can construct this object from just  e.g. "English"